src/bsoncxx/test/json.cpp

```diff
@@ -112,4 +112,25 @@ TEST_CASE("CXX-1246: Canonical Extended JSON") {
         output ==
         R"({ "number" : { "$numberInt" : "42" }, "bin" : { "$binary" : { "base64": "ZGVhZGJlZWY=", "subType" : "04" } } })");
 }
+
+TEST_CASE("UDL _bson works like from_json()") {
+    using namespace bsoncxx;
+
+    SECTION("_bson and from_json() return the same value") {
+        auto expected_value = from_json(k_valid_json);
+        auto actual_value = R"({ "a" : 1, "b" : 2.0 })"_bson;
+        REQUIRE(actual_value == expected_value);
```

Comment:
Can you add a test case from the ticket as well: `REQUIRE ("[1,2,3]" == from_json("[1,2,3]"))`